azkaban-web-server/src/main/java/azkaban/flowtrigger/quartz/FlowTriggerScheduler.java

```diff
@@ -163,6 +163,10 @@ public boolean resumeFlowTriggerIfPresent(final int projectId, final String flow
               flowId, flowTrigger, submitUser, quartzTriggers.isEmpty() ? null
               : quartzTriggers.get(0), isPaused, flow.isLocked());
         } catch (final Exception ex) {
+          if (QuartzScheduler.isSerializationBug(ex) && this.scheduler.enableSerializationHack()) {
```

Comment:
If this condition is not met, shall we just throw the exception rather than logging it?
@li-ygerchikov  Your thoughts?

Reply:
Logging it and setting `scheduledFlowTrigger = null` is the existing behavior. Do you think it should be changed?

Reply:
Yes, I would retain the existing functionality for the purpose of this PR, @jakhani. If you think we need to revise this behavior (which might be a perfectly valid suggestion), I would do it separately.

Reply:
@li-ygerchikov Sounds good. Thank you so much.
@acharneski Thank you for responding to my comment. It can be taken care of in another PR by us like @li-ygerchikov mentioned.